app/src/main/java/erevacation/com/userlist/ui/homescreen/list/ListFragment.kt

```diff
@@ -0,0 +1,13 @@
+package erevacation.com.userlist.ui.homescreen.list
+
+import android.os.Bundle
+import android.view.Menu
+import android.view.MenuInflater
+import erevacation.com.userlist.R
+import erevacation.com.userlist.basic.BasicFragment
+import erevacation.com.userlist.databinding.FragmentListBinding
```

Comment:
It looks like we are importing here some classes which we don't need in implementation.